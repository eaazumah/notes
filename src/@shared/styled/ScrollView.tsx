import styled from 'styled-components/native';

const ScrollView = styled.ScrollView`
  flex: 1;
  width: 100%;
  padding-horizontal: 18px;
  background-color: ${props => props.theme.backgroundColor};
`;

export default ScrollView;
