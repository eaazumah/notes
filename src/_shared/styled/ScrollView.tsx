import styled from 'styled-components/native';

const ScrollView = styled.ScrollView`
  flex: 1;
  background-color: ${props => props.theme.backgroundColor};
`;

export default ScrollView;
