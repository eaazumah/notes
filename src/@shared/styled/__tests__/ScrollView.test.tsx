import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import TestProvider from '../../../__testUtils__/Providers';
import ScrollView from '../ScrollView';

test('styled ScrollView works', () => {
  const tree = renderer
    .create(
      <TestProvider>
        <ScrollView />
      </TestProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
