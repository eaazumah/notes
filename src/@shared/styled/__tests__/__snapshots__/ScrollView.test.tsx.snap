// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styled ScrollView works 1`] = `
<RCTScrollView
  style={
    Array [
      Object {
        "backgroundColor": "#F5F5F5",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingHorizontal": 18,
        "width": "100%",
      },
    ]
  }
>
  <View />
</RCTScrollView>
`;
